src/main/java/org/apache/sling/discovery/commons/providers/spi/LocalClusterView.java

```diff
@@ -33,4 +38,21 @@ public String getLocalClusterSyncTokenId() {
         return localClusterSyncTokenId;
     }
 
+    public void setPartiallyStartedClusterNodeIds(Collection<Integer> clusterNodeIds) {
```

Comment:
There are no unit tests for these new methods. I'll create a PR for your feature branch.

Reply:
thx!